### ISSUE TITLE
Change a formula

### ISSUE THREAD

**Issue by ciola** [grady/miro-latex#4]
The refresh button does not work after changing the LaTeX string.
However, I found a workaround to change the formula:
1. Copy the string
2. Paste this string into the editor.
3. Modify the mathematical expression.
4. Send the modified formula to the MIRO board.
5. Delete the old formula.
It should be possible to implement these steps (except 3 of course).
What do you think?

**Comment by grady** [comment]
Yes, thanks @ciola, great idea. See the [discussion thread here](https://community.miro.com/ask-the-community-45/latex-and-math-editor-tool-8698) for discussion of a similar idea.

I have been thinking about how to implement editing. The method you present is fairly straightforward:
- User selects object
- Press button to import string from object title attribute.
- Edit string
- Press "replace selected image" button to delete the old image and place the new image in the same location.

The main drawback is that there isn't a clear way to restrict this operation to the images we create. (Does this even matter?)

The other idea I've had is to think about using the AppData facility (https://developers.miro.com/docs/board_board#getappdata) of the SDK, to store some sort of record of which images we've inserted into the board, so that we can scroll through them and only them. Would need to come up with some sort of compatible format (JSON?) to store and use relevant information to go this route.

Any thoughts on that?

**Comment by ciola** [comment]
Thank you for taking my comments into consideration.
Concerning the information to be provided, I have the impression that it is the following:
-LaTeX chain
- position (x, y, rotation angle)
- size
As for the colour(s), I managed to do it with LaTeX :
{\textcolor{#E6E6E6}{\lim_{x\to&#43;\infty}}{\textcolor{#FAC710}{f\left(x\right)}}\textcolor{#FF0000}{&#61;}{\textcolor{#F767D7}{l}}}

**Comment by ciola** [comment]
Hello Grady,
I read here https://community.miro.com/inspiration-and-connection-67/latex-and-math-editor-tool-8698
that it is now possible to edit a formula in the new version but I don't see the new button in my MIRO. Do I need to upgrade? If so how do I go about it? I thought that with MIRO the updates were automatic.

**Comment by grady** [comment]
> Hello Grady, I read here https://community.miro.com/inspiration-and-connection-67/latex-and-math-editor-tool-8698 that it is now possible to edit a formula in the new version but I don't see the new button in my MIRO. Do I need to upgrade? If so how do I go about it? I thought that with MIRO the updates were automatic.

@ciola This will only function for NEW equations that have been created since the feature was enabled (maybe a little over about a week ago?). And the button only shows up when you have exactly one such image selected while the panel is open.

Can you verify that both of these conditions are met?
